test/bench.js

```diff
@@ -135,9 +143,13 @@ function parseArg(argv) {
     } else if (arg[0] === '-') {
       if (arg.length > 2) {
         // e.g. -abc
-        argv = arg.substring(1).split('').map(ch => {
-          return `-${ch}`;
-        }).concat(argv);
+        argv = arg
+          .substring(1)
+          .split('')
+          .map((ch) => {
+            return `-${ch}`;
+          })
```

Comment:
```suggestion
          .map((ch) => `-${ch}`)
```

Nit pick: no need for `return` here